resource/reapi/bindings/c/reapi_module.cpp

```diff
@@ -48,7 +48,7 @@ extern "C" void reapi_module_destroy (reapi_module_ctx_t *ctx)
 }
 
 extern "C" int reapi_module_match (reapi_module_ctx_t *ctx,
-                                   const char *match_op, const char *jobspec,
+                                   enum match_op_t match_op, const char *jobspec,
```

Comment:
No need for `enum` type in function signature.
```suggestion
                                 match_op_t match_op, const char *jobspec,
```